validation/test_suite/2.0/should_pass/005_altids.json

```diff
@@ -7,47 +7,47 @@
 	"uri": "005_altids",
 	"altids": [
         {
-	        "name": "itemid",
+	        "role": "itemid",
```

Comment:
Looks good to me, just a minor formatting thing here around tab depth for the role vs value.